packages/bp/e2e/admin/ui.test.ts

```diff
@@ -4,6 +4,8 @@ import { bpConfig } from '../../jest-puppeteer.config'
 import { clickOn, expectMatch, fillField, uploadFile } from '../expectPuppeteer'
 import { closeToaster, expectAdminApiCallSuccess, expectCallSuccess, getResponse } from '../utils'
 
+const NEW_PASSWORD = '654321'
```

Comment:
Since this password absolutely needs to be different than the `bpConfig.password`, we should maybe generate a random string instead? This could also be done in the refactoring of the E2E tests!